packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -179,7 +182,7 @@ const blockTyping = (event: KeyboardEvent) => {
             type="button"
             :aria-label="t('select.toggleDropDown')"
             tabindex="-1"
-            :disabled="props.readonly || props.disabled || props.loading"
+            :disabled="disabled || props.loading"
```

Comment:
Did you remove readonly intentionally? Shouldn't the select be non-openable if its readonly?